Dockerfile

```
@@ -6,15 +6,17 @@ FROM node:22-alpine AS base
 
 WORKDIR /usr/src/app
 
-ENV YARN_DISABLE_GIT_HOOKS=1
 ENV CI=true
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
 
 RUN apk add --no-cache dumb-init g++ make python3
```

Hadolint (2.12.0)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)